test/test_subprocess.py

```diff
@@ -33,8 +33,12 @@ def test_timeout():
 sys.stderr.write("Stderr before waiting\n")
 sys.stdout.flush()
 sys.stderr.flush()
-subprocess.call([sys.executable, "-c",
-    "import sys, subprocess; subprocess.call([sys.executable, '-c', 'import time; time.sleep(360)'])"])
+subprocess.call([
+    sys.executable,
+    "-c",
+    "import sys, subprocess; subprocess.call("
+    "[sys.executable, -c', import time; time.sleep(360)'])"
```

Comment:
```suggestion
    "[sys.executable, '-c', import time; time.sleep(360)'])"
```
A quote is missing

Reply:
@datapythonista I fix the quote missing, but it is still not working.  Do you think I could use this code instead?

```
subprocess.call([sys.executable, "-c",
    "import sys, subprocess; subprocess.call(\
    [sys.executable, '-c', 'import time; time.sleep(360)'])"])
```

Reply:
The backslash is irrelevant, and as I mentioned, a discouraged practice in Python. There is another quote missing:
```python
[sys.executable, '-c', import time; time.sleep(360)']
```

```python
[sys.executable, '-c', 'import time; time.sleep(360)']
```